.circleci/config.yml

```diff
@@ -268,9 +266,8 @@ jobs:
       - capture_test_logs
 
   acceptanceTestsQuorum:
-    executor: quorum_ats_executor_xl
+    executor: quorum_ats_executor_med
     steps:
-      - prepare
```

Comment:
Is there a good reason the prepare step is removed?

Reply:
I don't think we need it. this CI just uses the develop docker image which is already built elsewhere